### PR TITLE
tests: add attachement tests integration

### PR DESCRIPTION
**Pull request by sapk**
Extracted from #7956, to make them work before changing (rebasing) anything.

I haven't very much time lately to do a rebase of the PR in one-shoot. I think this could be already merged as it add more test on current code and it will ease the review later when I rebase the PR with fix.

### CODE REVIEW

File: integrations/attachement_test.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+	if expectedStatus != http.StatusOK {
+		return ""
+	}
+	var obj map[string]string
+	DecodeJSON(t, resp, &obj)
+	return obj["uuid"]
+}
+
+func TestCreateAnonymeAttachement(t *testing.T) {
```
Comment (guillep2k) [inline integrations/attachement_test.go]:
```suggestion
func TestCreateAnonymousAttachment(t *testing.T) {
```

**Review by guillep2k — APPROVED** [review]
Typo nit. Otherwise LGTM.

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
wait https://github.com/go-gitea/gitea/pull/9309#discussion_r355796649

File: integrations/attachement_test.go (hunk 1 above)
Comment (sapk) [reply integrations/attachement_test.go]: Good catch thx.

**Comment by sapk** [comment]
@lunny done

**Pull request merged**